src/main.js

```diff
@@ -52,33 +60,52 @@ function displayCurrentCover() {
 
 function displayFormView() {
   homeViewSection.classList.add("hidden");
-  savedViewSection.classList.add("hidden"); 
-  formViewSection.classList.remove("hidden");  
-  randomCoverButton.classList.add("hidden"); 
+  savedViewSection.classList.add("hidden");
+  formViewSection.classList.remove("hidden");
+  randomCoverButton.classList.add("hidden");
   saveCoverButton.classList.add("hidden");
   homeButton.classList.remove("hidden");
 //*************ADDED code BELOW to ITER2 ********
   viewSavedButton.classList.remove("hidden");
+  makeYourOwnButton.classList.add("hidden");
+  //hide makeyourownbutton
```

Comment:
Despite this not being in the directions for the project, it just makes logical sense!  Why have a button to navigate you to the page you are currently on?  I agree with this addition!